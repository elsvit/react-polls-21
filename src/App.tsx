/**
 * @fileOverview Root App component
 */

import React, {ReactElement} from 'react';
import {Provider} from 'react-redux';

import {store} from './store';
import AppRouter from './AppRouter';

import './app.scss';

const App = (): ReactElement => (
  <Provider store={store}>
    <div className="app">
      <AppRouter />
    </div>
  </Provider>
);

export default App;