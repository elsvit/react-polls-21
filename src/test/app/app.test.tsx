import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import App from '../../App';

describe('App', () => {
  it('should render without error', () => {
    const Renderer = ShallowRenderer.createRenderer();

    Renderer.render(<App />);

    const wrapper = Renderer.getRenderOutput();

    expect(wrapper).toMatchSnapshot();
  });
});