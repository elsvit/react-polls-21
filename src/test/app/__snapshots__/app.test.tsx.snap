// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without error 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <div
    className="app"
  >
    <AppRouter />
  </div>
</Provider>
`;
